inc/mqttclient.class.php

```diff
@@ -146,16 +158,27 @@ public function publish($topic, $message, $qos = 0, $retain = 0) {
       return false;
    }
 
+   /**
+    * Disconnects the MQTT client according to the ping response
```

Comment:
This method is a bit tricky. I used the ping response event to break the infinite loop implemented in the MQTT client library. This is the best solution I found without patching it.